ena_upload/templates/ENA_template_experiments.xml

```diff
@@ -19,7 +29,9 @@
                         <PAIRED py:when="row.library_layout == 'paired'" NOMINAL_LENGTH="${row.insert_size}" />
                         <SINGLE py:when="row.library_layout == 'single'" />
                     </LIBRARY_LAYOUT>
+                    <py:if test="attributetest(row, 'library_construction_protocol')">
```

Comment:
While `library_construction_protocol` and `design_description` are optional in the SRA.experiment.xsd and in the webin submissions portal, at least one is mandatory if an RNA-seq experiment is being submited. At least according to the [support guides](https://ena-docs.readthedocs.io/en/latest/exp_01.html?#rna-seq-experiment-attributes-to-include-in-your-experiment-objects)

I am not sure how it is validated when submitting through the webin submissions portal, since it lists both fields as optional.

Reply:
ok great thanks!